pkg/virt-launcher/virtwrap/api/schema_test.go

```diff
@@ -182,6 +182,84 @@ var exampleXMLppc64le = `<domain type="kvm" xmlns:qemu="http://libvirt.org/schem
   <iothreads>2</iothreads>
 </domain>`
 
+// TODO: Make the XML fit for real arm64 configuration
```

Comment:
what do you plan to change here?

Reply:
For arm64 arch, some configuration are not supposed to be in the xml, e.g. we are using uefi boot rather bios boot.